lib/jekyll-pandoc.rb

```diff
@@ -1,2 +1,31 @@
-require File.expand_path("../jekyll/converters/markdown.rb", __FILE__)
-require 'jekyll/converters/markdown/pandoc_parser'
\ No newline at end of file
+class Jekyll::Converters::Markdown::PandocProcessor
+  DEFAULT_EXTENSIONS = []
+  DEFAULT_FORMAT = 'html5'
+
+  def initialize(config)
+    require 'pandoc-ruby'
+    @config = config
+  rescue LoadError
+    STDERR.puts 'You are missing a library required for Pandoc. Please run:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.